src/test/java/TaskTest.java

```diff
@@ -0,0 +1,19 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TaskTest {
+    @Test
+    public void testStringConversion() {
+        assertEquals("[T][ X ] eat apples", new Todo("eat apples").toString());
+    }
+
+    @Test
+    public void getStatusIcon_todo_ok() {
```

Comment:
Invalid function name

Reply:
Test functions should start with the word "test" and use camelCase instead of underlined_function_names